edx_rest_framework_extensions/permissions.py

```diff
@@ -83,21 +90,61 @@ def has_permission(self, request, view):
         of the organizations specified in the content provider filters
         in the JWT used to authenticate.
         """
-        if _should_skip_jwt_scope_enforcement(request):
-            return True
         course_key = CourseKey.from_string(view.kwargs.get('course_id'))
         jwt_filters = decode_jwt_filters(request.auth)
-        for provider_type, filter_value in jwt_filters:
-            if provider_type == 'content_org' and filter_value == course_key.org:
+        for filter_type, filter_value in jwt_filters:
+            if filter_type == 'content_org' and filter_value == course_key.org:
                 return True
+        log.warning("Permission JwtHasContentOrgFilterForRequestedCourse not satisfied.")
```

Comment:
Same logging detail question.  Would it be useful to know about details of an org mismatch for example?

Reply:
I added the org name in the warning message.